### DISCUSSION TITLE
Use unknown instead of any in *MapTo operators

### DISCUSSION THREAD

**Discussion by krokosik** [ReactiveX/rxjs#6713] [Report issues other than bug]
Currently operators like `mapTo`, `mergeMapTo` and so on, have a type of `OperatorFunction<any, T>`, which does not cause any errors on its own, but introduces an unsafe `any` argument, which is picked up by this [ESLint rule](@typescript-eslint/no-unsafe-argument). Changing it to `unknown`, is a rather cosmetic fix, but It fixes the problem and is more in line with how TypeScript should be used.

**Comment by kwonoj** [comment]
https://github.com/ReactiveX/rxjs/issues/6399

Core team have long term planning to deprecate *To variant eventually. If current eslint rule is blocking, recommendation is use `map(() => )` instead.

**Comment by Wyzard256** [comment]
This was recently resolved by #6738.
